views/js/checkout-component-renderer.js

```diff
@@ -504,6 +496,40 @@ jQuery(document).ready(function() {
             }
         }
 
+        function handleOnClick(resolve, reject) {
+            const paymentMethodContainer = getFormPaymentMethodContainer(this.paymentForm);
+            const paymentMethodType = paymentMethodContainer.data('local-payment-method');
+            // Show message if button is disabled else if not in progress, hide and resolve
+            if (prestaShopPlaceOrderButton.prop('disabled') && !isPlaceOrderInProgress()) {
+                showRequiredConditionsInfoMessage(paymentMethodContainer);
+                if (paymentMethodType === 'paypal') {
```

Comment:
pls add a TODO to remove the paypal extra logic and only use the resolve/reject when the component fixes it (no Focal number for it yet)